api/controllers/v1/jobs.ts

```diff
@@ -106,7 +108,9 @@ export const FailStuckJobs = async () => {
   try {
     const hours = 8;
     await jobRepository.failStuckJobs(hours);
+    client.close();
   } catch (err) {
+    client.close();
```

Comment:
(Nit suggestion; non-blocking) Since we're intending to close in both cases, it might be a good practice to wrap this in a `finally` block. This can also be applied in a few other places

Reply:
👀 Just wrapped it - good catch!